ocs_ci/helpers/helpers.py

```diff
@@ -2790,3 +2790,24 @@ def fetch_used_size(cbp_name, exp_val=None):
             f"matching. Retrying"
         )
     return used_in_gb
+
+
+def get_mon_pdb():
```

Comment:
sorry for not raising it earlier, but how about making this function generic, to get any type of pod, such as OSD PDB, MDS, etc.

What do you think?

Reply:
I see this PR already address this, https://github.com/red-hat-storage/ocs-ci/pull/1260/files Do you still need this function to be made generic ? Please correct me if my understanding is wrong

Reply:
so maybe we can merge these 2 together into a single function that gets the pod type and returns the PDB?

Reply:
ACK

Reply:
this could be done separately, please open an issue to track the needed change

Reply:
https://github.com/red-hat-storage/ocs-ci/issues/3634